nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/access-policies/feature/access-policies.component.html

```diff
@@ -15,14 +15,14 @@
   ~  limitations under the License.
   -->
 
-<div class="pb-5 flex flex-col h-screen justify-between gap-y-5">
-    <header class="nifi-header">
+<div class="flex flex-col h-screen justify-between">
+    <header class="pb-5 nifi-header">
         <navigation></navigation>
     </header>
     <div class="px-5">
-        <h3 class="primary-color">Access Policies</h3>
+        <h3 class="pb-5 primary-color">Access Policies</h3>
     </div>
-    <div class="px-5 flex-1">
+    <div class="pb-5 px-5 flex-1">
         <router-outlet></router-outlet>
     </div>
 </div>
```

Comment:
What are these changes doing? It looks like it would result in the same layout without them.

Reply:
It is very similar but these changes do align the bottom padding/spacing with the other pages. Specifically the spacing for the last refreshed timestamp and button.

Reply:
I don't believe these changes make any difference in the padding/spacing. And the changes are different from the other pages. They do not impact the refreshed timestamp and button. Is there something subtle that I missed?

Reply:
Sorry, I must have not been looking at main. I do agree that these changes do not impact the refreshed timestamp and button. I have reverted this file.